portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,25 @@
+body {
+    background-color: #212529;
+    color: #fff;
+}
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  /* margin-left: auto; */
```

Comment:
Similar to my comment on the other review, it would be preferable to remove these rather than comment them out. The original values can be found via git history.

Reply:
Got it. I am working on a massive upgrade to the portfolio. Comments will be removed in the next pull requests.